logging_manager/src/StorageManager.py

```diff
@@ -0,0 +1,59 @@
+# Library imports
+import os
+import shutil
+import psutil
+import time
+
+def storage_manager_main():
+    storage_directory = "/media/files"
+
+    # Read data from tmp directory
+    tmp_path = "/tmp"
+
+    for files in os.listdir(tmp_path):
+        file_path = os.path.join(tmp_path, files)
+
+        if os.path.isfile(file_path) and files.endswith(".json"):
+            print(files)
+            with open(file_path, 'r') as file:
+                read_data = file.read()
+                print(read_data)
+
+        else:
+            print("File not found")
```

Comment:
This is going to print for every file that isnt a .json, "File not Found" doesn't really make sense. The file was found, its just not a .json